lib/client/ipam_block.go

```diff
@@ -361,11 +361,10 @@ func getIPVersion(ip cnet.IP) ipVersion {
 	return ipv4
 }
 
-func largerThanBlock(blockCIDR cnet.IPNet) bool {
-	ones, bits := blockCIDR.Mask.Size()
-	prefixLength := bits - ones
```

Comment:
prefixLength is just `ones` when you do `bits - ones`, for example for subnet of `/24` it will return prefixLength of `8`  which is incorrect, it should be `24`
https://play.golang.org/p/D2a2bKhOdE